ycms/cms/models/patient.py

```diff
@@ -115,6 +187,29 @@ def current_ward(self):
         """
         return self.current_room.ward if self.current_room else None
 
+    def _get_date_status(self, date_to_check):
+        """
+        Helper function to get the date status of a given date
```

Comment:
```suggestion
    @staticmethod
    def _get_date_status(date_to_check):
        """
        Helper function to get the date status of a given date
```
since this is not dependent on any object attributes/methods